test/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+ROOTDIR=$(git rev-parse --show-toplevel)
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+RF_VENV="${ROOTDIR}/_output/robotenv"
+
+if [ ! -d "${RF_VENV}" ]; then
+    python3 -m venv "${RF_VENV}"
+    "${RF_VENV}/bin/python3" -m pip install -r "${SCRIPTDIR}/requirements.txt"
+fi
```

Comment:
Can we add this functionality to `scripts/fetch_tools.sh` script to make it consistent with the rest of checks?

Reply:
Great point, done.